components/Header/Header.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
```

Comment:
rename file name to **index.tsx**
You can remove duplicate naming in the import. for example 'Header/Header'

Reply:
fixed